examples/v3.0/uspto.yaml

```diff
@@ -0,0 +1,216 @@
+openapi: 3.0.0
+servers:
+  - url: 'https://developer.uspto.gov/ibd-api'
```

Comment:
Is it worth turning the scheme into a variable with enums of `http` and `https`?
If not, what about adding a description to explain the two URLs?

Reply:
I must confess I hadn't thought of using server variables for anything except the host, port or path (where RFC6570 templates are normally used), but happy to go with that, or the description option.

Reply:
I don't have a strong pref, either will improve this slightly. Given that http/https should be somewhat common, it seems like that is self-evident as a variable.